vms/platformvm/block/executor/verifier_test.go

```diff
@@ -1184,8 +1184,9 @@ func TestBlockExecutionWithComplexity(t *testing.T) {
 			}
 
 			require.Contains(verifier.blkIDToState, blkID)
-			onAcceptState := verifier.blkIDToState[blkID].onAcceptState
-			require.Equal(test.expectedFeeState, onAcceptState.GetFeeState())
+			blockState := verifier.blkIDToState[blkID]
+			require.Equal(blk, blockState.statelessBlock)
```

Comment:
I don't know enough about what's going on to give a thoughtful review of all this. Would this have otherwise failed?

I'm looking for something in the tests that indicates to someone knowledgable about this code that there's been a proper fix here.

Reply:
> Would this have otherwise failed?

Yes.

`require.Equal(blk, blockState.statelessBlock)` fails on master and passes on this branch.

`require.Equal(&blk.ApricotStandardBlock, blockState.statelessBlock)` would have passed on master.

Reply:
In that case, LGTM

> [!CAUTION]
> Once you start using GitHub Markdown alerts, you can't stop.